src/main/java/pixelperfect/route/AsteroidNode.java

```diff
@@ -0,0 +1,24 @@
+package pixelperfect.route;
+
+/**
+ * An asteroid field Node which belongs in a Route. The asteroid field will be a dangerous location
+ * to fly in.
+ * 
+ * @author Floris Doolaard
+ *
+ */
+public class AsteroidNode extends RouteNode {
+
+  /**
+   * The constructor for the AsteroidNode.
+   * 
+   * @param summary
+   *          , a summary of the AsteroidNode.
+   * @param description
+   *          , a description of the AsteroidNode.
+   */
+  public AsteroidNode(String summary, String description) {
+    super(summary, description);
```

Comment:
Ok, so you want to have fixed String values in the subtypes and have setters rather than having it in RouteNodes? Do you think it will make a huge difference (technically)?